packages/flutter_test/lib/src/binding.dart

```diff
@@ -1104,6 +1111,14 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
       _timeout += duration;
   }
 
+  @override
+  Future<T> executeLater<T>(Duration duration, FutureOr<T> body()) async {
+    assert(_currentFakeAsync != null);
+    addTime(duration);
+    _currentFakeAsync.elapse(duration);
+    return await body();
+  }
```

Comment:
This isn't so much "Execute a callback in a later time for the binding" so much as "advance the clock and execute the callback". Why not just have people use `pump`, which also advances the clock?

Reply:
The idea here is this method is for the same thing, we do it differently: for Live binding we wait for a real period of time, but for the binding with a fake clock, we add time and then perform the callback. The `addTime` method only add timeout but don't add the fake clock time. The pump schedule a frame which is not preferred here.